subprojects/docs/src/transforms/release-notes.gradle

```diff
@@ -32,10 +32,10 @@ transformDocument {
             append("<title>Gradle @version@ Release Notes</title>").
             append("<link rel='stylesheet' type='text/css' href='https://assets.gradle.com/lato/css/lato-font.css'/>")
 
-    head().append("<style>p{}</style>").children().last().childNode(0).attr("data", baseStyleFile.text + releaseNotesStyleFile.text)
+    head().append("<style>${baseStyleFile.text}${releaseNotesStyleFile.text}</style>")
```

Comment:
Do you know what changed and why we can't use the `data` attribute anymore?

Reply:
I don't. But it started doing nothing from jsoup 1.11.1. I didn't find anything related in the release notes https://jsoup.org/news/release-1.11.1